### ISSUE TITLE
How to use primewords and st-cmd data sets

### ISSUE THREAD

**Issue by lampardwk** [audier/DeepSpeechRecognition#70]
Hi！Thank you so much for sharing such great code！I have some problems，Primewords and st-cmd data are not divided into training sets and test sets，Is it necessary to divide into training set, Validation set and test set by myself ? Looking forward to your reply, thank you！
